templates/web/pages/tag/tag.tt.html

```diff
@@ -48,6 +50,42 @@
             [% description %]
         </div>
     [% END %]
+    [% IF tagid.defined  %]
+        <!-- injecting facet-knowledge-panel -->
+        <div id="facet-knowledge-panel">
+            <h2 id="title"></h2>
+            <P id="facet-panels"></P>
+        </div>
+        <!-- Fetching facet knowledge panel -->
+        <script>
+            let facet_kp_url = "https://facets-kp.openfoodfacts.net/render-to-html?";
+            let params = "facet_tag=[% tagtype_name %]&value_tag=[% tagid %]&sec_facet_tag=[% tagtype2_name %]&sec_value_tag=[% tagid2 %]";
+            // checking if data is filtered on country 
+            if ("[% country_code %]" !== "world"){
+                // Adding language code to get transalted data from facet-knowledge-panel API
+                // Removing langcode from county name l.e, "en:france" to "france"
+                country_name = "[% country %]".slice(3);
+                params += "&lang_code=[% country_code %]&country=" + country_name;
```

Comment:
matching on country name instead of country code sounds very strange... 

Why not pass the 2 letter country code + the 2 letter language code?  &lang_code=[% lc %]&country_code=[% cc ]

On that note, I think it would be great to keep the same field names in the different platforms and projects. We use "lc" and "cc" extensively, the less renaming we have, the easiest it is.

Reply:
I can do this, little tweak in the fcaet_kp side will solve this :-)